src/views/Settings.vue

```diff
@@ -194,6 +194,21 @@
             <ion-toggle :checked="configurePicker" @ionChange="setConfigurePickerPreference($event)" slot="end" />
           </ion-item>
         </ion-card>
+
+        <ion-card>
+          <ion-card-header>
+            <ion-card-title>
+              {{ $t("Partial Order rejection") }}
+            </ion-card-title>
+          </ion-card-header>
+          <ion-card-content>
+            {{ $t('Specify whether you reject partial BOPIS order when any order item’s inventory is insufficient at the store.') }}
+          </ion-card-content>
+          <ion-item lines="none">
+            <ion-label>{{ $t("Allow partial rejection") }}</ion-label>
+            <ion-toggle :checked="partialOrderRejection" @ionChange="setPartialOrderRejectionPreference($event)" slot="end" />
```

Comment:
This setting has already been implemented and merged. So please remove this and the related code.